bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -412,9 +412,47 @@ def _canJsDisplay(self):
 
 
     def _getJsCode(self):
+        # Add extra primitives to canvas with custom colors, palette, gStyle
+        prim = self.drawableObject.GetListOfPrimitives()
+
+        style = ROOT.gStyle
+        colors = ROOT.gROOT.GetListOfColors()
+        palette = 0
```

Comment:
This is ok, but it would be more pythonic to set `palette` here to `None`, and later check `if palette is not None`. The same comment applies to `style` and `colors`, which are set to `0` below too.